accesscontroller/default.go

```diff
@@ -1,3 +1,4 @@
+// Default access controller for IPFS Log
```

Comment:
Maybe you should make it clear that it's a noop access controller? (in addition to the CanAppend comment)